packages/ui/src/apis/register.ts

```diff
@@ -15,23 +15,23 @@ export const register = async (username: string, password: string) => {
     .json<Response>();
 };
 
-export const sendPhonePasscode = async (phone: string) => {
+export const sendSMSPasscode = async (phone: string) => {
```

Comment:
camelcase `registerSendSmsPasscode` and etc.

<img width="687" alt="image" src="https://user-images.githubusercontent.com/10594507/163101751-ba0f9ed6-ed2d-47b7-bda2-e5d7d9656af8.png">

See Notion doc: [coding convention](https://www.notion.so/silverhand/Common-36449f7172514044967e5dbe537b87e9#3deb4db5eb7846b28ce2d61a376a0082)

Reply:
@IceHe updated PTAL